components/function-controller/Gopkg.toml

```diff
@@ -1,72 +1,50 @@
 required = [
-    "github.com/emicklei/go-restful",
-    "github.com/onsi/ginkgo", # for test framework
-    "github.com/onsi/gomega", # for test matchers
-    "k8s.io/client-go/plugin/pkg/client/auth/gcp", # for development against gcp
-    "k8s.io/code-generator/cmd/client-gen", # for go generate
-    "k8s.io/code-generator/cmd/deepcopy-gen", # for go generate
-    "sigs.k8s.io/kustomize",
     "sigs.k8s.io/controller-tools/cmd/controller-gen", # for crd/rbac generation
-    "sigs.k8s.io/controller-runtime/pkg/client/config",
-    "sigs.k8s.io/controller-runtime/pkg/controller",
-    "sigs.k8s.io/controller-runtime/pkg/handler",
-    "sigs.k8s.io/controller-runtime/pkg/manager",
-    "sigs.k8s.io/controller-runtime/pkg/runtime/signals",
-    "sigs.k8s.io/controller-runtime/pkg/source",
-    "sigs.k8s.io/testing_frameworks/integration", # for integration testing
-	  "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
+    "k8s.io/code-generator/cmd/deepcopy-gen", # for go generate
+    "sigs.k8s.io/kustomize", # for easy deployment by developers
 ]
 
-
 [prune]
   go-tests = true
 
 [[override]]
-  name="k8s.io/apiextensions-apiserver"
-  version="kubernetes-1.13.4"
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.13.4"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version="kubernetes-1.13.4"
+  version = "kubernetes-1.13.4"
 
 [[override]]
   name = "k8s.io/api"
-  version="kubernetes-1.13.4"
+  version = "kubernetes-1.13.4"
```

Comment:
should we follow the kubernetes version from knative 0.8?

Reply:
That's the topic of another PR: #5560 (currently in review)